packages/amplify_core/lib/src/types/auth/attribute/cognito_user_attribute_key.dart

```diff
@@ -66,7 +66,6 @@ class CognitoUserAttributeKey extends UserAttributeKey
   List<Object?> get props => [
         // Cognito will lowercase these in API calls
         key.toLowerCase(),
-        readOnly,
```

Comment:
Why are we removing this?

Reply:
You'd never have two attributes with the same key but different readOnly markers so it doesn't provide any additional info beyond just the key. Further, AWSDebuggable uses these properties to render `toString` and it doesn't add any value there.